resources/views/components/atoms/_input.blade.php

```diff
@@ -10,7 +10,7 @@
     @if (isset($textCount) and $textCount)
     <span class="input__io-container" data-behavior="textCount">
     @endif
-    <input class="f-secondary" type="{{ $type ?? 'text'}}" value="{{ $value ?? '' }}" id="{{ $id ?? '' }}" name="{{ $name }}" placeholder="{{ $placeholder ?? '' }}" {{ $disabled ?? '' }} {{ (isset($pattern)) ? 'pattern=' .$pattern : '' }} {{ $required ?? '' }}>
+    <input class="f-secondary" type="{{ $type ?? 'text'}}" value="{{ $value ?? '' }}" id="{{ $id ?? '' }}" name="{{ $name }}" {{ (isset($placeholder)) ? 'placeholder=' .$placeholder : '' }} {{ (isset($autocomplete)) ? 'autocomplete=' .$autocomplete : '' }} {{ $disabled ?? '' }} {{ (isset($pattern)) ? 'pattern=' .$pattern : '' }} {{ $required ?? '' }}>
```

Comment:
@web-dev-trev taking a closer look at this line, I think there’s a layer of quotes that would be helpful to add here:  

```
{{ (isset($autocomplete)) ? 'autocomplete=“' .$autocomplete .’”’ : '' }}
```

Note the added double quote after the autocomplete attribute name, and the concatted end quote after its value. On the off chance there’s a space in the `$attribute` value, this would generate output that looks like this:

```
autocomplete=“value value value”
```

Instead of 

```
autocomplete=value value value 
```

Could you make this change for both autocomplete and placeholder in a separate PR?